app/controllers/requests/received_requests_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Requests::ReceivedRequestsController < ApplicationController
   def index
-    @requests = current_user.received_requests
+    @requests = current_user.received_requests.sort_by { |request| request.duedate }
```

Comment:
I found it. thank you!
http://guides.rubyonrails.org/active_record_querying.html#ordering